downstream/titles/release-notes/topics/docs-24.adoc

```diff
@@ -0,0 +1,26 @@
+// This is the release notes for AAP 2.4 documentation, the version number is removed from the topic title as part of the release notes restructuring efforts.
+
+[[docs-2.4-intro]]
+= Ansible Automation Platform Documentation
+
+The documentation set for Red Hat Ansible Automation Platform 2.4 has had significant updates to improve the experience for our customers and the Ansible community. 
+
+
+.Enhancements
+
+* With the removal of the on-premise component Automation Services Catalog from Ansible Automation Platform 2.4 onwards, the Automation Services Catalog documentation too is removed from the Ansible Automation Platform 2.4 documentation.
```

Comment:
, all Automation Services catalog documentation is removed...

Reply:
Made the change.